This test appears to fail in GNU Guix and elsewhere.  It has been reported
upstream at https://github.com/puma/puma/issues/995

diff --git a/spec/concurrent/channel_spec.rb b/spec/concurrent/channel_spec.rb
index d70fba8..4f29a8b 100644
--- a/spec/concurrent/channel_spec.rb
+++ b/spec/concurrent/channel_spec.rb
@@ -598,7 +598,7 @@ module Concurrent
           }.to raise_error(ArgumentError)
         end
 
-        it 'loops until the block returns false' do
+        xit 'loops until the block returns false' do
           actual = 0
           expected = 3
           latch = Concurrent::CountDownLatch.new(expected)
